WTraining/pull_request_template.md

```diff
@@ -1,11 +1,13 @@
 ## Summary
```

Comment:
This file is only a template. Github take this template and automatically use it in the description of the PR, then you put your comments there